routemaster/config/visualisation.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
```

Comment:
It looks like this file is rendered with Jinja2? Could we perhaps encode that in the filename?

Reply:
I'd also quite like to introduce a linter for this if we can.

https://pypi.org/project/html-linter/ looks quite good, although because this isn't strictly HTML, it's Jinja2, we might have trouble with it in the future. I can't find anything nice for Jinja.